discovery.md

```diff
@@ -224,11 +229,47 @@ The following sections define the attributes that appear in a Service entity.
   See the Primer for more information.
 
 - Constraints:
-  - REQUIRED
+  - conditionally REQUIRED. See above.
   - MUST be a valid UUID per RFC4122.
 - Examples:
   - `bf5ff5cc-d059-4c79-a89a-2513e45a1340`
 
+##### epoch
+
+- Type: `Integer`
+- Description: The Discovery Endpoint's epoch value for this Service Entry.
+  This specification does not mandate any particular semantic meaning to
+  the value used. For example, implementations are free to use a value that
+  represents a timestamp or could choose to simply use a monotomically
+  increasing number. The only requirement is that the value MUST always
+  increase each time the Service Entry is updated. This allows for a quick
+  integer comparision to determine which version of this Service Entry is the
+  latest - meaning, the one with the larger integer value.
+
+  While this attibute is REQUIRED to be part of the the Service's description
+  when returned from a Discovery Endpoint, depending on the action from the
+  client it might not be mandatory to appear in a client's request. For a
+  "create" operation, this attribute MUST NOT appear. For an "import" type
```

Comment:
There is no create in the discovery spec.

Reply:
Yep I didn't think I'd find a good way to talk about it in the management API stuff but I did, so I removed this text now.